bootstrapping_tools/bootstrapping.py

```diff
@@ -157,8 +285,17 @@ def mean_bootstrapped(data, N=2000):
     return np.squeeze(bootstrap_mean)
 
 
-def remove_outlier(method, lambdas_bootstraps, **kwargs):
+def remove_outlier(method, data, **kwargs):
+    """Function to select method to filter the outliers in data.
```

Comment:
Creo que el _"Function"_ sale sobrando:

```suggestion
    """Select method to filter the outliers in data.
```